tests/test_toa_writer.py

```diff
@@ -42,10 +42,14 @@ def test_roundtrip_topo_toa_Tempo2format(self):
         # Create a barycentric TOA
         t1time = Time(58534.0, 0.0928602471130208, format="mjd", scale="utc")
         t1 = toa.TOA(t1time, obs="gbt", freq=0.0)
-        ts = toa.get_TOAs_list([t1], ephem="DE421")
+        t2 = toa.TOA(t1time, obs="gbt", freq=0.0)
+        ts = toa.get_TOAs_list([t1, t2], ephem="DE421")
+        assert ts.ntoas == 2
+        ts.table[1]["flags"]["ignore"] = True  # comment TOA
         ts.write_TOA_file("testtopo.tim", format="Tempo2")
         ts2 = toa.get_TOAs("testtopo.tim")
         print(ts.table, ts2.table)
+        assert ts2.ntoas == 1  # one should be commented
```

Comment:
Can you make this a separate test, so we can tell which things are broken and which are not?

Reply:
Done